### ISSUE TITLE
build script Babel Polyfill-exclusions not found

### ISSUE THREAD

**Issue by djcowan** [WordPress/gutenberg#65479]
### Description

While running @wordpress/scripts we are receiving an error when calling  start or build 
- `[BABEL]: Cannot find module './polyfill-exclusions'
 current package dev/dependencies:
1. `"@wordpress/interactivity": "^6.8.0"` 
2. `"@wordpress/scripts": "^30.0.0",`
3.  "typescript": "^5.5.4"
4. `update to op` - `"@wordpress/url": "^4.8.0"`

### Step-by-step reproduction instructions

The plugin uses a viewModuleScript and interactivity: true
1. in plugin root - run script `"packages-update": "wp-scripts packages-update"`
2. run `"wp-scripts build --experimental-modules",` or `"wp-scripts build",`
___
We will continue to review our setup

### Screenshots, screen recording, code snippet

```ssh
ERROR in ./src/dialog/view.ts
Module build failed (from ./node_modules/babel-loader/lib/index.js):
Error: [BABEL]: Cannot find module './polyfill-exclusions'
Require stack: 
```
our tsconfig
```json
{
	"$schema": "https://json.schemastore.org/tsconfig.json",
	"compilerOptions": {
		"forceConsistentCasingInFileNames": true,
		"allowJs": true,
		"checkJs": true,
		"allowSyntheticDefaultImports": true,
		"jsx": "preserve",
		"target": "esnext",
		"module": "esnext",
		"lib": [
			"DOM",
			"DOM.Iterable",
			"ESNext"
		],
		"declaration": true,
		"declarationMap": true,
		"composite": true,
		"emitDeclarationOnly": true,
		"isolatedModules": true,
		"skipDefaultLibCheck": true,
		/* Strict Type-Checking Options */
		"strict": true,
		/* Additional Checks */
		"noImplicitReturns": true,
		"noFallthroughCasesInSwitch": true,
		/* Module Resolution Options */
		"moduleResolution": "node",
		/* This needs to be false so our types are possible to consume without setting this */
		"esModuleInterop": false,
		"resolveJsonModule": true,
		"typeRoots": [
			"./node_modules/@types"
		],
		"types": []
	},
	"include": [
		"./src",
		"./src/*"
	],
	"exclude": [
		"**/*.android.js",
		"**/*.ios.js",
		"**/*.native.js",
		"**/benchmark",
		"packages/*/build-*/**",
		"packages/*/build/**",
		"**/test/**",
		"packages/**/react-native-*/**"
	]
}
```

### Environment info

Gutengerg Version 19.2.0
package versions as listed
NB unable to confirm:
`Please confirm that you have tested with all plugins deactivated except Gutenberg.`

### Please confirm that you have searched existing issues in the repo.

- [X] Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

- [X] Yes

**Comment by gregdev** [comment]
I'm guessing this is related to https://github.com/WordPress/gutenberg/pull/65292

**Comment by djcowan** [comment]
thanks will read.
`update`
I have just run a fresh create-block. But I'm running in a dev container which might be tainted.
Will exit and run again on bare metal.  
```ssh
npx @wordpress/create-block@latest todo-list
Need to install the following packages:
@wordpress/create-block@4.51.0
// ...

./src/index.js 39 bytes [built] [code generated] [1 error]

ERROR in ./src/index.js
Module build failed (from ./node_modules/babel-loader/lib/index.js):
Error: [BABEL]: Cannot find module './polyfill-exclusions'
```

**Comment by gregdev** [comment]
@djcowan it looks like there are two files, `polyfill-exclusions.js` and `replace-polyfills.js`, that are missing from the @wordpress/babel-preset-default package. As a local workaround you can comment out lines 9-10 from `node_modules/@wordpress/babel-preset-default/index.js` until a fix is released.

**Comment by namjul** [comment]
I think this is still an issue.

In https://github.com/WordPress/gutenberg/pull/65292/files `@wordpress/babel-preset-default` got 2 new files like [polyfill-exclusions.js](https://github.com/WordPress/gutenberg/pull/65292/files#diff-31bad49926fb27f4838943284c38b1a16826f9af2be3316c2492b290cb364ae9).
But it's missing in the published version https://www.npmjs.com/package/@wordpress/babel-preset-default?activeTab=code.

We might want to add  these to package.json fields .

**Comment by gregdev** [comment]
Actually for a better workaround: specifying your own babel config file will prevent WordPress from providing its own configuration, so the `@wordpress/babel-preset-default` package won't be loaded.

In my project I've created a `.babelrc.js` file which seems to work for my purposes:

```
const { hasArgInCLI } = require('@wordpress/scripts/utils')

const isProduction = process.env.NODE_ENV === 'production'
const hasReactFastRefresh = hasArgInCLI('--hot') && !isProduction

module.exports = {
    presets: [require.resolve('@babel/preset-typescript')],
    plugins: [
        require.resolve('@wordpress/warning/babel-plugin'),
        [
            require.resolve('@babel/plugin-transform-react-jsx'),
            {
                runtime: 'automatic',
            },
        ],
        hasReactFastRefresh && require.resolve('react-refresh/babel'),
    ].filter(Boolean),
}

```

**Comment by djcowan** [comment]
Thank you @gregdev for looking into the issue and for providing a useful patch. Greatly appreciated. 
@namjul - thank you for confirming the issue. I closed the ticket after being made aware of #65292 
I could not be sure the issue was not the result of a local misconfiguration

**Comment by sirreal** [comment]
An updated version of the affected packages has been released so this issue should be addressed without requiring workarounds.
